public/global.css

```diff
@@ -46,3 +46,21 @@ a {
 .lock-screen {
   cursor: wait;
 }
+
+.transition {
```

Comment:
`transition` might be to generic as a class name. How about `button-transition`?